packages/portal/src/Portal/Portal.types.ts

```diff
@@ -4,6 +4,7 @@ import { OneOf } from '@leafygreen-ui/lib';
 
 export type PortalProps = {
   children?: React.ReactNode;
+  _ref?: React.MutableRefObject<HTMLElement | null>;
```

Comment:
not sure if `_ref` is the best prop name (the underscore makes it seem like it's unused or internal)

Reply:
I wanted to name this `ref` but it would create a console warning because the `Portal` component wasn't wrapped in a `forwardRef`. `_ref` seems appropriate since it is internally used and the `portalRef` prop is what we expose to external users. Do you have an alternative name suggestion?

Reply:
ok updated it to `portalRef`